war/src/main/js/add-item.js

```diff
@@ -0,0 +1,414 @@
+       * 'jenkins.branch.OrganizationFolder.org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigator' ] },
+       */
+    {
+      order : 7,
+      id : 'copy',
+      display : 'Copy existing item',
+      placement:'special-single',
+      minToShow:0,
+      description : 'Use an existing item as a starting point for creating a new item configuration.',
+      instances : [
+        'copy'
+      ]
+    },
+    {
+      order : 3,
+      id : 'templates',
+      display : 'Template creators',
+      description : 'Items that allow for the parameerized creation of reusable core system items or components of those items.',
+      remainders:'basic',
+      instances : [
+          'com.cloudbees.hudson.plugins.modeling.impl.jobTemplate.JobTemplate', 
+          'com.cloudbees.hudson.plugins.modeling.impl.folder.FolderTemplate',
+          'com.cloudbees.hudson.plugins.modeling.impl.auxiliary.AuxModel', 
+          'com.cloudbees.hudson.plugins.modeling.impl.builder.BuilderTemplate',
+          'com.cloudbees.hudson.plugins.modeling.impl.publisher.PublisherTemplate'
+
+      ]
+    }, {
+      order : -4,
+      id : 'user-templates',
+      display : 'Custom templates',
+      placement:'special-group',
+      minToShow:1,
+      parents : [
+          {
+            id : 'folderTemplate',
+            display : 'Folder template',
+            description : 'A template for creating parameterized folders',
+            parent : 'com.cloudbees.hudson.plugins.modeling.impl.jobTemplate.JobTemplate'
+          }, {
+            id : 'jobTemplate',
+            display : 'Job template',
+            description : 'A template for creating parameterized jobs',
+            parent : 'com.cloudbees.hudson.plugins.modeling.impl.folder.FolderTemplate'
+          }
+      ],
+      instances : function() {
+      }
+    }
+];
+
+$jq(function() {
+  
+  var $ = $jq;
+  var jRoot = $('head').attr('data-rooturl');
+  var defaultMinToShow = 2;
+  var $root = $jq('#main-panel');
+  var $form = $root.find('form[name="createItem"]').addClass('jenkins-config');
+  var $table = $form.find('table.createItem');
+  var $newView = $jq('<div class="new-view" />').insertBefore($form);
+  var $tabs = $('<div class="jenkins-config-widgets" />').appendTo($newView);
+  var $categories = $('<div class="categories" />').appendTo($newView);
+  var sectionsToShow = [];
+
+  function sortItemsByOrder(itemTypes) {
+    function sortByOrder(a, b) {
+      var aOrder = a.order;
+      var bOrder = b.order;
+      return ( (aOrder < bOrder) ? -1 : ( (aOrder > bOrder) ? 1 : 0));
+    }
+
+    return itemTypes.sort(sortByOrder);
+  }
+
+  function getCategory(i, elem) {
+    var category = {
+      id : 'j-add-item-type-' + elem.id,
+      display : elem.display,
+      description : elem.description,
+      minToShow : elem.minToShow,
+      remainders : elem.remainders,
+      items : []
+    };
+    if (typeof i === 'string')
```

Comment:
Aamoi ... are you running `gulp` after making these changes? That will run tests, do lint checks etc.